packages/ses/src/whitelist-intrinsics.js

```diff
@@ -285,6 +285,15 @@ export default function whitelistIntrinsics(
           delete obj[prop];
         } catch (err) {
           if (prop in obj) {
+            if (typeof obj === 'function' && prop === 'prototype') {
+              obj.prototype = undefined;
+              if (obj.prototype === undefined) {
+                // eslint-disable-next-line @endo/no-polymorphic-call
+                console.warn(`Tolerating undeletable ${subPath} === undefined`);
```

Comment:
Will this not always warn?

Reply:
Hmm. Unless the `prototype: 'undefined'` property is explicitly defined in the whitelist?

Reply:
In any case, for present purposes, I prefer that it always warn.

Reply:
Oh, I see the context.